src/platform/packages/shared/kbn-unified-tabs/src/components/tabs_bar/tabs_bar.tsx

```diff
@@ -36,12 +38,14 @@ export const TabsBar: React.FC<TabsBarProps> = ({
   maxItemsCount,
   tabContentId,
   getTabMenuItems,
+  services,
   onAdd,
   onLabelEdited,
   onSelect,
   onClose,
 }) => {
-  const { euiTheme } = useEuiTheme();
+  const euiThemeContext = useEuiTheme();
+  const { euiTheme } = euiThemeContext;
```

Comment:
Out of curiosity, what's the purpose of that change?

Reply:
Ah I see below that it's used `getTabsShadowGradient`. In this file it's not used anywhere though, right? Or it's just for consistency?

Reply:
I think it was an intermediate change during the implementation. I will revert it here, thanks!